maker/src/routes.rs

```diff
@@ -108,12 +108,10 @@ pub struct CfdNewOfferParamsRequestDeprecated {
     pub min_quantity: Usd,
     pub max_quantity: Usd,
     pub tx_fee_rate: Option<TxFeeRate>,
-    /// The _daily_ funding rate as decided upon by the caller.
-    ///
-    /// The fact that this is the _daily_ funding rate is part of the API contract between this
-    /// application and its caller. Changing this would be a breaking change.
-    #[serde(rename = "funding_rate")]
-    pub daily_funding_rate: Option<FundingRate>,
+    /// The current _daily_ funding rate for the maker's long position
+    pub daily_funding_rate_long: Option<FundingRate>,
+    /// The current _daily_ funding rate for the maker's short position
+    pub daily_funding_rate_short: Option<FundingRate>,
```

Comment:
Don't change this in the deprecated one please, it will break the automation.

Reply:
Right, the depricated API can only add, but not change values that were previously there.